.gitignore

```diff
@@ -1,2 +1,4 @@
 vendor/
 composer.lock
+.php_cs.cache
+.idea
```

Comment:
.idea should not be here

Reply:
(also think to new space at end of file)

Reply:
Ok, .idea removed and empty line added